openstack/loadbalancer/v2/flavorprofiles/requests.go

```diff
@@ -0,0 +1,110 @@
+package flavorprofiles
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// LIST
+
+type ListOptsBuilder interface {
+	ToFlavorProfileListQuery() (string, error)
+}
+
+type ListOpts struct {
+	Fields []string `q:"fields"`
```

Comment:
Could you add documentation for all the fields? for example
```suggestion
	// The fields that you want the server to return
	Fields []string `q:"fields"`
```